application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/rest/MultipleControllersEndpointIT.java

```diff
@@ -35,6 +35,38 @@
     private static final String ROLES_URL = "/api/user-roles/%s";
     private static final String WORKBASKET_INPUT_DEFINITION_URL = "/api/display/work-basket-input-definition/%s";
     private static final String JURISDICTIONS_URL = "/api/data/jurisdictions";
+    private static final String CASE_TYPE_URL = "/api/data/case-type/%s";
+
+    @Test
+    public void shouldReturnCaseType() throws Exception {
```

Comment:
Don't we have a IT test already that retrieves a CaseType that we can update instead of adding a brand new? Asking because IT tests are slow

Reply:
I checked and we do not have a full e2e test for get CaseType. There is a class CaseDefinitionControllerTest that is using MockMvc but is also using mocks so is not truly e2e.